versioned_docs/version-5/system.mdx

```diff
@@ -21,6 +21,16 @@ const actor = interpret(machine).start();
 actor.system;
 ```
 
+The root of a system can also be explicitly assigned a `systemId` in the `interpret` function:
+
+```ts
+import { interpret } from 'xstate';
+
+const actor = interpret(machine, { systemId: 'root-id' }).start();
+```
+
+This is useful for actors in the system to be able send events to the root actor.
+
```

Comment:
Unrelated, but the "Coming soon", below, could use some other styling to make it stand out a bit more. @laurakalbag Any ideas for how we could improve those "Coming soon" instances?

Reply:
Most of the **coming soon…**s are todos in disguise 😆 But ones that are teasing Studio features could always go in a studio admonition box like:

```
:::studio

Coming soon… systems in Stately Studio’s editor 🤫 

:::
```

(Emoji is a bit of fun)